app/pages/homepage/context.js

```diff
@@ -1,9 +1,9 @@
 import content from './manifest.json';
 import { buyingCatalogueAdminHost, orderFormHost } from '../../config';
 
-export const getHomepageContext = ({ user }) => ({
+export const getHomepageContext = ({ req }) => ({
   ...content,
-  showAdminTile: !!(user && user.organisation && user.organisation.toLowerCase() === 'manage'),
+  showAdminTile: !!(req.user && req.user.organisation && req.user.organisation.toLowerCase() === 'manage'),
```

Comment:
Do we need to change this to receive the full `req` object? It looks like we're still only using the `user` object.

Reply:
it is passed in via the routes.js which uses 2 other methods which are shared and require the req object in various places.  To decouple the object for just this method is painfull and creates a mess of code

Reply:
fixed